### PR TITLE
go to rel, go directly to rel, do not pass go, do not collect $200

### PR DESCRIPTION
**Pull request by anurse**
/cc @piotrpMSFT @pranavkm @davidfowl @brthor @Sridhar-MS

### CODE REVIEW

File: src/Microsoft.DotNet.Tools.New/FSharp_Console/project.json [hunk 1]
```diff
@@ -1,4 +1,4 @@
-﻿{
+{
     "version": "1.0.0-*",
     "compilationOptions": {
         "emitEntryPoint": true
```
Comment (Sridhar-MS) [inline src/Microsoft.DotNet.Tools.New/FSharp_Console/project.json]: What about this file - https://github.com/dotnet/cli/pull/1077/files#diff-e933ad30dec8519bdbbe45d501906a85R14?

File: src/Microsoft.DotNet.Cli.Utils/project.json [hunk 2]
```diff
@@ -6,9 +6,9 @@
     },
 
     "dependencies": {
-        "NETStandard.Library": "1.0.0-rc3-23727",
+        "NETStandard.Library": "1.0.0-rc2-23727",
```
Comment (pranavkm) [inline src/Microsoft.DotNet.Cli.Utils/project.json]: Is it intentionally not `*`-ed?

File: src/Microsoft.DotNet.Tools.New/FSharp_Console/project.json (hunk 1 above)
Comment (anurse) [inline src/Microsoft.DotNet.Tools.New/FSharp_Console/project.json]: Missed by my script :). Good catch!

File: src/Microsoft.DotNet.Cli.Utils/project.json (hunk 2 above)
Comment (anurse) [inline src/Microsoft.DotNet.Cli.Utils/project.json]: Yes, we're avoiding `*` dependencies to ensure we remain on a constant build.

**Comment by anurse** [comment]
FYI, this is blocking some ASP.NET build scenarios /cc @pranavkm 

We're going to be waiting for #1086 to go in since that will be a major blocker /cc @davidfowl @piotrpMSFT

**Comment by anurse** [comment]
@dotnet-bot test this please

**Comment by livarcocc** [comment]
:shipit:

**Comment by piotrpMSFT** [comment]
@anurse, just rebased atop rel/1.0.0

**Pull request merged**